shared/utils/src/roundValue.js

```diff
@@ -73,3 +74,17 @@ export function roundValue2(value) {
 	if (abs > 0.01) return Number(value.toFixed(4)) // 0.012345 to 0.0123
 	return value // as is
 }
+
+/** Use to return displayed values in scientific notation
+ * Do not use for values intented for calculation later.
+ */
+export function formatValue(value, digits) {
+	const v = Number(value)
+	if (Number.isInteger(v)) return v
+	const abs = Math.abs(v)
+	if (abs < 1 || abs > 9999) {
+		//Number() reverts positive values less than 10^21 to a whole number
+		return abs > 9999 ? v.toPrecision(digits) : Number(v.toPrecision(digits))
```

Comment:
should be `Number(v.toPrecision(digits))` it doesn't make sense to return string or number in different cases

Reply:
I added a comment in [6c25f4d](https://github.com/stjude/proteinpaint/pull/2439/commits/6c25f4d3a9ca5b584010b5d3a3aad0d0d2fe761b)